applications/asset_tracker/overlay-rsa.conf

```diff
@@ -0,0 +1,2 @@
+CONFIG_BOOT_SIGNATURE_TYPE_RSA=y
```

Comment:
I suggest to rename the file to `mcuboot_overlay-rsa.conf` to make it visible that this overlay file is intended for the mcuboot image.